YouVsVirus/Assets/Scripts/Components/LevelStats.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System;
+using UnityEngine;
+
+/// <summary>
+/// The statistics for the currently active level.
+/// Counts how many humans are infected/dead/immun etc.
+/// The GameObject this script is attached to will be kept across the scene change.
+/// </summary>
+public class LevelStats : MonoBehaviour
+{
+    /// <summary>
+    /// The Number of NPCs that were spawned in the level
+    /// </summary>
+    private int NumberOfNPCs;
+
+    /// <summary>
+    /// Total count of exposed NPCs.
+    /// </summary>
+    private int NumberExposed = 0;
+
+    /// <summary>
+    /// Total count of infected NPCs.
+    /// </summary>
+    private int NumberInfected = 0;
+
+    /// <summary>
+    /// Total count of dead NPCs.
+    /// </summary>
+    private int NumberDead = 0;
+
+    /// <summary>
+    /// Total count of recovered NPCs.
+    /// </summary>
+    private int NumberRecovered = 0;
+
+    /// <summary>
+    /// Tracks whether the init function was called.
+    /// Used for debugging.
+    /// </summary>
+    private bool isInit = false;
+
+    /// <summary>
+    /// Initialize with initial number of NPCs and initial number
+    /// of infected.
+    /// </summary>
+    public void Init(int InitialNumberOfNPCs, int NumberOfInitiallyInfected)
+    {
+        // Set initiali numbers of NPCs and infected humans
+        NumberOfNPCs = InitialNumberOfNPCs;
+        NumberInfected = NumberOfInitiallyInfected;
+        // At the beginning only the infected humans are exposed
+        NumberExposed = NumberOfInitiallyInfected;
```

Comment:
The initially infected are counted twice: Once in this line, and once in HumanBase's `Start()` method when `SetCondition(_initialCondition)` is called. I guess we solve this by deleting this line?

Reply:
I fixed it now.